src/test/java/uk/co/ramp/covid/simulation/population/ShiftsTest.java

```diff
@@ -14,13 +14,14 @@
 
 public class ShiftsTest {
 
-    @Ignore("Failing Test")
     @Test
     public void testGetShift() throws JsonParseException, IOException {
         ParameterReader.readParametersFromFile("src/test/resources/default_params.json");
 
        Shop shop = new Shop(CommunalPlace.Size.LARGE);
        int end = shop.getShifts().getShift(1).getEnd();
+       assertEquals(15, end);
+       end = shop.getShifts().getShift(1).getEnd();
```

Comment:
Not sure I understand how this works but both asserts pass now

Reply:
Shifts is an infinitely long queue of possible shift patterns (for when we assign workers) for a given day. For shops the first shift you get is 8-15, then 15-22, 8-15, 15-22, and so on.